ocw/lib/k8s.py

```diff
@@ -0,0 +1,16 @@
+from datetime import datetime, timezone
+
+
+def cleanJobs(provider, client, cluster_name):
```

Comment:
```suggestion
def cleanJobs(provider : Provider, client : BatchV1Api , cluster_name: str):
```

you need to correct this but idea is to use python typing